opentelemetry/src/metrics/instruments/mod.rs

```diff
@@ -72,6 +76,12 @@ impl<'a, T> HistogramBuilder<'a, T> {
         self.unit = Some(unit.into());
         self
     }
+
+    /// Set the boundaries for this histogram.
```

Comment:
// Setting boundaries is optional. By default, the boundaries are set to
        // [0.0, 5.0, 10.0, 25.0, 50.0, 75.0, 100.0, 250.0, 500.0, 750.0, 1000.0, 2500.0, 5000.0, 7500.0, 10000.0]

please copy this wording too.